lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -1520,7 +1520,7 @@ namespace Js
 
     ScriptFunctionType * FunctionProxy::EnsureDeferredPrototypeType()
     {
-        Assert(this->GetFunctionProxy() == this);
+        //Assert(this->GetFunctionProxy() == this);
```

Comment:
Okay.

I haven't really been able to work out what that assert is for, and it seems to be possible to make it happen without the other changes in this hotfix. (See #1062)